packages/mermaid/src/diagrams/class/classDb.ts

```diff
@@ -40,30 +40,18 @@ let functions: any[] = [];
 
 const sanitizeText = (txt: string) => common.sanitizeText(txt, getConfig());
 
-const splitClassNameAndType = function (_id: string) {
-  const id = common.sanitizeText(_id, getConfig());
+const splitClassIdAndType = function (_id: string) {
+  const id = sanitizeText(_id);
   let genericType = '';
-  let className = id;
+  let classId = id;
 
   if (id.indexOf('~') > 0) {
     const split = id.split('~');
-    className = sanitizeText(split[0]);
+    classId = sanitizeText(split[0]);
     genericType = sanitizeText(split[1]);
   }
 
-  return { className: className, type: genericType };
-};
-
-export const setClassLabel = function (_id: string, label: string) {
-  const id = common.sanitizeText(_id, getConfig());
-  if (label) {
-    label = sanitizeText(label);
-  }
-
-  const { className } = splitClassNameAndType(id);
-  classes.get(className)!.label = label;
-  classes.get(className)!.text =
-    `${label}${classes.get(className)!.type ? `<${classes.get(className)!.type}>` : ''}`;
+  return { classId: classId, type: genericType };
```

Comment:
nit

```suggestion
  return { classId, type: genericType };
```

Reply:
I tried that at first, but lint compained